pkgs/by-name/ch/chawan/package.nix

```diff
@@ -0,0 +1,70 @@
+{ lib
+, stdenv
+, fetchFromSourcehut
+, makeBinaryWrapper
+, curlMinimal
+, man
+, ncurses
+, nim
+, pandoc
+, perl
+, pkg-config
+, zlib
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "chawan";
+  version = "20240301";
+
+  src = fetchFromSourcehut {
+    owner = "~bptato";
+    repo = "chawan";
+    rev = "87ba9a87be15abbe06837f1519cfb76f4bf759f3";
+    hash = "sha256-Xs+Mxe5/uoxPMf4FuelpO+bRJ1KdfASVI7rWqtboJZw=";
+    fetchSubmodules = true;
+  };
+
+  buildInputs = [ curlMinimal ncurses zlib ];
+  nativeBuildInputs = [
+    makeBinaryWrapper
+    nim
+    pandoc
+    pkg-config
+  ];
+
+  postPatch = ''
+    # patch perl shebangs in the w3mman replacement;
+    # this script is used when displaying man pages
+    # using mancha or cha, given a 'man:' URL
+    substituteInPlace adapter/protocol/man \
```

Comment:
i think if you add perl to nativeBuildInputs patchShebangsAuto should automatically pick it up

Reply:
It does, done